tests/chainer_tests/training_tests/extensions_tests/test_parameter_statistics.py

```diff
@@ -0,0 +1,113 @@
+import re
+import unittest
+
+import mock
+import numpy
+import six
+
+import chainer
+from chainer import testing
+from chainer.training import extensions
+
+
+@testing.parameterize(
+    {'statistics': {'min': numpy.min},
+     'links': chainer.links.Linear(10, 10),
+     'expect': 4},
+    {'statistics': {'mean': numpy.min, 'std': numpy.std},
+     'links': chainer.links.Linear(10, 10),
+     'expect': 8},
+    {'statistics': extensions.ParameterStatistics.default_statistics,
+     'links': chainer.links.Linear(10, 10),
+     'expect': 48},
+    {'statistics': None,
+     'links': chainer.links.Linear(10, 10),
+     'expect': 0}
+)
+class TestParameterStatistic(unittest.TestCase):
+
+    def setUp(self):
+        self.trainer = mock.MagicMock()
+        self.reporter = chainer.Reporter()
+
+    def test_report(self):
+        extension = extensions.ParameterStatistics(self.links,
+                                                   statistics=self.statistics)
+        with self.reporter:
```

Comment:
I think we can leave it as it is since you need to instantiate a `Report` anyhow.